packages/CodeQuality/tests/Rector/Foreach_/ForeachItemsAssignToEmptyArrayToAssignRector/Fixture/skip_false_positive_1.php.inc

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Rector\CodeQuality\Tests\Rector\Foreach_\ForeachItemsAssignToEmptyArrayToAssignRector\Fixture;
+
+class SkipFalsePositive1
```

Comment:
Explicit name would be better, when someone will try to debug this rule in the future.

E.g. `skip_non_empty_dim_key.php.inc`

Reply:
@TomasVotruba Done!